client/state/jetpack-product-install/actions.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Internal dependencies
+ */
+import {
+	JETPACK_PRODUCT_INSTALL_REQUEST,
+	JETPACK_PRODUCT_INSTALL_STATUS_RECEIVE,
+	JETPACK_PRODUCT_INSTALL_STATUS_REQUEST,
+} from 'state/action-types';
+
+import 'state/data-layer/wpcom/jetpack-blogs/product-install';
+import 'state/data-layer/wpcom/jetpack-blogs/product-install-status';
+
+/**
+ * Start the Jetpack product install process for that site.
+ *
+ * @param {Number}  siteId        The ID of the site.
+ * @param {String}  akismetKey    Akismet key.
+ * @param {Number}  vaultpressKey VaultPress key.
+ * @return {Object}               Action object.
+ */
+export const startJetpackProductInstall = ( siteId, akismetKey, vaultpressKey ) => ( {
```

Comment:
Minor arch nit, it's easy to forget the order of key args here, and it wouldn't scale that well to moar keys. What about changing the sig to `siteId, { akismet, vaultpress }`?

Reply:
Well, that's surely intentional on my side. If you forget the order, you better get an error :) The idea is that I want to force those keys to be required, and having them as arguments makes sure of that, at least more than having an object with the keys. I also prefer the clear signature without nested arguments.

Reply:
Also, FWIW I'm hopeful we won't need to set VP up anytime soon, as we transition to Jetpack Backups. That will leave us with only AK, and therefore additional nesting will be obsolete.

Reply:
I tend to agree with @ockham, but I also think this is unlikely to introduce any real problems ever, so I'm not especially concerned either way 🙂